backstop/tests/carousel.html

```diff
@@ -215,95 +98,145 @@ <h2>Default</h2>
 
           <button
             class="iui-button iui-borderless iui-small"
+            type="button"
             aria-label="Pause animation"
-            data-action="pause"
```

Comment:
This change breaks the script bellow. Change data-action back or update script.

Reply:
Thank you, this is fixed now.